CI/scripts/test.sh

```diff
@@ -202,22 +202,39 @@ function test-mtls-ssl-client-server-comm-pytest() {
 
     cd pytests
 
+    # ----
     # Exercise client-server communication with self-signed certificates
     pytest --capture=tee-sys -v test_client_server_mtls.py -k test_server_process_with_mtls_self_signed_cert &
 
     sleep 5
 
     pytest --capture=tee-sys -v test_client_server_mtls.py -k test_client_app_with_mtls_self_signed_cert
 
+    # ----
     # Basic test to exercise core certificate verification v/s root-CA-cert
     pytest --capture=tee-sys -v test_client_server_mtls.py -k test_verify_certs_versus_root_cert
 
+    # ----
     # Exercise client-server communication with root-CA-signed certificates
-    pytest --capture=tee-sys -v test_client_server_mtls.py -k test_server_process_with_mtls_root_signed_cert &
+    pytest --capture=tee-sys -v test_client_server_mtls.py \
+            -k test_server_process_with_mtls_root_signed_cert_and_pvt_key_file &
 
     sleep 5
 
-    pytest --capture=tee-sys -v test_client_server_mtls.py -k test_client_app_with_mtls_root_signed_cert
+    pytest --capture=tee-sys -v test_client_server_mtls.py \
+            -k test_client_app_with_mtls_root_signed_cert_and_pvt_key_file
+
+    # ----
+    # Exercise client-server communication with root-CA-signed certificates
+    # using private-key written-to and read-from a temp-file, for certificate
+    # verification.
+    pytest --capture=tee-sys -v test_client_server_mtls.py \
+            -k test_server_process_with_mtls_root_signed_cert_using_temp_pvt_key_file &
```

Comment:
New test case which will write out private-key data to a named temporary file, in an anonymous (but named) temp-dir.